projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.e2e.ts

```diff
@@ -411,27 +409,42 @@ describe('dot-key-value', () => {
     describe('@Methods', () => {
         beforeEach(async () => {
             element.setAttribute('name', 'fieldName');
-            element.setAttribute('required', true);
+            element.setAttribute('value', 'first key|first value,second key|second value');
+
             spyValueChangeEvent = await page.spyOnEvent('valueChange');
             spyStatusChangeEvent = await page.spyOnEvent('statusChange');
         });
 
         describe('reset', () => {
-            it('should clear the field and emit', async () => {
-                element.setAttribute('value', 'first key|first value,second key|second value');
+            it('should clear the field and emit invalid (field required)', async () => {
+                element.setAttribute('required', true);
+                await page.waitForChanges();
```

Comment:
s this needed?

Reply:
Yes because in the previous line I'm doing setAttribute.

Reply:
what I mean is that if you remove https://github.com/dotCMS/core-web/blob/ab1c16107b3817f7178664629efe7ef8b508c23e/projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.e2e.ts#L421

it still works!